### ISSUE TITLE
Ability to setup mtu for network interface

### ISSUE THREAD

**Issue by negative-creep90** [Telmate/terraform-provider-proxmox#410]
Hi! sometimes I've need to change mtu at network interfaces. is it possible to add ability to change mtu on a network interface? this feature is available with proxmox 6.2.3 an later.
│ Error: Unsupported argument
│ 
│   on modules/vm/main.tf line 55, in resource "proxmox_vm_qemu" "vm_qemu":
│   55:       mtu       = network.value.mtu
│ 
│ An argument named "mtu" is not expected here.

**Comment by negative-creep90** [comment]
in this doc information about mtu exists - https://pve.proxmox.com/wiki/Qemu/KVM_Virtual_Machines 
you can do this at /etc/pve/qemu-server/100.conf at net0 or any other net
at this forum i've https://forum.proxmox.com/threads/set-mtu-via-cloud-init.72426/ that minuminm supported version is qemu-server: 6.2-3

**Comment by BaldFabi** [comment]
Ok, I didn't expect that this option isn't available through the gui. It maybe could be implemented into the provider if the Proxmox api provides it.

The Telmate Proxmox api lib doesn't return the mtu which would be necessary for the provider.

```json
  "network": {
    "0": {
      "bridge": "vmbr0",
      "id": 0,
      "macaddr": "5A:5D:C7:B8:74:1B",
      "model": "virtio"
    }
  }
```

And as far as I seen, not even the api itself provides the field ([vm config](https://pve.proxmox.com/pve-docs/api-viewer/index.html#/nodes/{node}/qemu/{vmid}/config) reference).

**Comment by negative-creep90** [comment]
https://pve.proxmox.com/pve-docs/api-viewer/index.html#/nodes/{node}/qemu mtu setting is available. Thanks

**Comment by BaldFabi** [comment]
Direct linking within the api viewer doesn't work. Where did you find it? Even if I follow the path in your url I can't find it.

**Comment by negative-creep90** [comment]
![image](https://user-images.githubusercontent.com/29099232/136079999-fe415485-7bdf-42c4-8d2f-3b2e5362299a.png)
i've captured screen with address and path of API

**Comment by BaldFabi** [comment]
That's really weird, I just see this
![image](https://user-images.githubusercontent.com/62034751/136089830-e3aaf627-8d02-455b-ae62-4d0e94bbc4f0.png)
That's the reason I couldn't find it.

**Comment by negative-creep90** [comment]
please change tab to "POST" from "Get" at 2nd colomn ender "Path: "

**Comment by BaldFabi** [comment]
Yes, you are right. Expected the same value listing in all methods

**Comment by BaldFabi** [comment]
I actually don't know. I'm just a random contributer 😄
Hopefully someone from Telmate will see this

**Comment by h0tw1r3** [comment]
I believe this was addressed in #498 and #499
